run-build.sh

```diff
@@ -39,4 +39,6 @@ echo "Executing user command: $cmd"
 eval "$cmd"
 CODE=$?
 
+cache_artifacts
```

Comment:
why are you adding this?

Reply:
I don't know, I'm not the author. Should I remove it?

Reply:
yeah, i'd remove it

Reply:
Fixed in e4e7354

Reply:
Ah funny I have re-added this!

I know why the author added this. It's because when you run the test, the cache is restored (by `install_dependencies` in line 29) but it is not saved after the job.